src/group_theory/group_action/quotient.lean

```diff
@@ -288,4 +290,49 @@ begin
   exact fintype.of_equiv _ (quotient_group.quotient_ker_equiv_range _).symm.to_equiv,
 end
 
+lemma stabilizer_conj_act_eq_centralizer (g : G) :
+  mul_action.stabilizer (conj_act G) g = (zpowers g).centralizer :=
+le_antisymm (le_centralizer_iff.mp (zpowers_le.mpr (λ x, mul_inv_eq_iff_eq_mul.mp)))
+  (λ x h, mul_inv_eq_of_eq_mul (h g (mem_zpowers g)).symm)
```

Comment:
I don't think this belongs here? Maybe in `group_theory.group_action.conj_act` instead?